managing-os/templates/archlinux.md

```diff
@@ -16,27 +16,666 @@ be considered experimental as Qubes developers team use mainly Fedora-based VMs
+
+
+<br>
+<br>
+
+
+    # /etc/pacman.conf
+    #
+    # See the pacman.conf(5) manpage for option and repository directives
+
+    #
+    # GENERAL OPTIONS
+    #
+    [options]
+    # The following paths are commented out with their default values listed.
+    # If you wish to use different paths, uncomment and update the paths.
+    #RootDir     = /
+    #DBPath      = /var/lib/pacman/
+    #CacheDir    = /var/cache/pacman/pkg/
+    #LogFile     = /var/log/pacman.log
+    GPGDir      = /etc/pacman.d/gnupg/
+    HoldPkg     = pacman glibc
+    #XferCommand = /usr/bin/curl -C - -f %u > %o
+    #XferCommand = /usr/bin/wget --passive-ftp -c -O %o %u
+    #CleanMethod = KeepInstalled
+    #UseDelta    = 0.7
+    Architecture = auto
+
+
+    # Pacman won't upgrade packages listed in IgnorePkg and members of IgnoreGroup
+    #IgnorePkg   =
+    #IgnoreGroup =
+    #NoUpgrade   =
+    NoUpgrade = /etc/X11/xinit/xinitrc.d/pulseaudio
+    NoUpgrade = /etc/X11/xinit/xinitrc.d/pulseaudio
+    NoUpgrade = /etc/X11/xinit/xinitrc.d/pulseaudio
+    #NoExtract   =
+
+    # Misc options
+    #UseSyslog
+    #Color
+    #TotalDownload
+    CheckSpace
+    #VerbosePkgLists
+
+    # By default, pacman accepts packages signed by keys that its local keyring
+    # trusts (see pacman-key and its man page), as well as unsigned packages.
+**Edited Line:** `#SigLevel    = Required DatabaseOptional`
+
+    LocalFileSigLevel = Optional
+    #RemoteFileSigLevel = Required
+
+    # NOTE: You must run `pacman-key --init` before first using pacman; the local
+    # keyring can then be populated with the keys of all official Arch Linux
+    # packagers with `pacman-key --populate archlinux`.
+
+    #
+    # REPOSITORIES
+    #   - can be defined here or included from another file
+    #   - pacman will search repositories in the order defined here
+    #   - local/custom mirrors can be added here or in separate files
+    #   - repositories listed first will take precedence when packages
+    #     have identical names, regardless of version number
+    #   - URLs will have $repo replaced by the name of the current repo
+    #   - URLs will have $arch replaced by the name of the architecture
+    #
+    # Repository entries are of the format:
+    #       [repo-name]
+    #       Server = ServerName
+    #       Include = IncludePath
+    #
+    # The header [repo-name] is crucial - it must be present and
+    # uncommented to enable the repo.
+    #
+
+    # The testing repositories are disabled by default. To enable, uncomment the
+    # repo name header and Include lines. You can add preferred servers immediately
+    # after the header, and they will be used before the default mirrors.
+
+    #[testing]
+    #SigLevel = PackageRequired
+    #Include = /etc/pacman.d/mirrorlist
+
+    [core]
+**Edited Line:** `SigLevel = PackageRequired`
```

Comment:
Those lines folds into preceding line... Maybe a blank line before?
And generally - why not set `SigLevel = PackageRequired` as default (and override when needed)?